usermods/audioreactive/audio_reactive.h

```diff
@@ -2060,5 +2105,5 @@ const char AudioReactive::_analogmic[]  PROGMEM = "analogmic";
 #endif
 const char AudioReactive::_digitalmic[] PROGMEM = "digitalmic";
 const char AudioReactive::_addPalettes[]       PROGMEM = "add-palettes";
-const char AudioReactive::UDP_SYNC_HEADER[]    PROGMEM = "00002"; // new sync header version, as format no longer compatible with previous structure
+const char AudioReactive::UDP_SYNC_HEADER[]    PROGMEM = "AUD02"; // new sync header version, as format no longer compatible with previous structure
```

Comment:
Maybe I've missed something, but I do not see any structural changes to the sync packet. Breaking compatibility between existing installs, with MM and well as other 3rd party software that send V2 data presumably  just to use a value that lets you know this ESP now data is for the audio usermod is really bad

It would be better surely to wrap the UM specific data inside another object that you send over ESP-NOW?

Reply:
> Breaking compatibility between existing installs, with MM and well as other 3rd party software that send V2 data presumably just to use a value that lets you know this ESP now data is for the audio usermod is really bad

Agreed, though I find 6 bytes, just to differentiate (audio) packet version, a wasteful decision.

> It would be better surely to wrap the UM specific data inside another object that you send over ESP-NOW?

It already is a packet within packet, I do not like the idea of yet another packet within. What we can do is to offset audio packet within `buffer.data[]` and prepend "Audio" "flag". I'll do that and revert audio packet header.

Reply:
I fully agree with @netmindz. This change will make lots of audio-sending tools (not just other WLED  variants) unusable.